src/constructs/loadbalancing/clb.ts

```diff
@@ -0,0 +1,32 @@
+import type { CfnLoadBalancer, LoadBalancerProps } from "@aws-cdk/aws-elasticloadbalancing";
+import { LoadBalancer } from "@aws-cdk/aws-elasticloadbalancing";
+import type { GuStack } from "../core";
+
+enum DeletionOverrideProperties {
+  SCHEME = "Scheme",
+}
+
+interface GuClassicLoadBalancerProps extends LoadBalancerProps {
+  overrideId?: boolean;
+  deletionOverrideProperties?: DeletionOverrideProperties[];
```

Comment:
Similarly, `propertiesToRemove` and `customProperties`/`bespokeProperties` would make a little more sense (to me) here.